src/nodes/not.jl

```diff
@@ -0,0 +1,5 @@
+export NOT
+
+struct NOT end
+
+@node NOT Deterministic [out, in1]
```

Comment:
rename to `in`

Reply:
And adjust in future parts of the code (e.g. rules and tests)

Reply:
We use it in the multiplication node I believe

Reply:
True!